wolfcrypt/src/asn.c

```diff
@@ -26849,12 +26868,15 @@ static int DecodeAsymKeyPublic(const byte* input, word32* inOutIdx, word32 inSz,
     *pubKeyLen = inSz - *inOutIdx;
     XMEMCPY(pubKey, input + *inOutIdx, *pubKeyLen);
 #else
+    word32 len = inSz - *inOutIdx;
```

Comment:
Some compilers will complain that `len` is being declared in the middle of the function. Please move to top of function.

Reply:
Fixed.